pkg/webui/console/components/events/previews/downlink-message.js

```diff
@@ -52,7 +51,6 @@ const DownLinkMessagePreview = React.memo(({ event }) => {
       const rx2DataRateIndex = getByPath(data, 'request.rx2_data_rate_index')
       return (
         <DescriptionList>
-          <DescriptionList.Byte title={messages.gatewayEUI} data={gatewayEUI} />
```

Comment:
I don't really understand why this was added in the first place. If you see the live data of a gateway, you know which gateway you are seeing, so providing the gateway EUI is really weird.